### PR TITLE
Add FastSpeech setup guide in README

### PR DESCRIPTION
**Pull request by ryangawei**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,50 @@
...
+5. Download model file and create output folders
+    Follow <https://github.com/roedoejet/NEU-tts-2022#neu-quickstart-instructions>.
+
+6. Synthesize
+
+   ```bash
+   python3 synthesize.py --text "YOUR_DESIRED_TEXT" --restore_step 900000 --mode single -p config/LJSpeech/preprocess.yaml -m config/LJSpeech/model.yaml -t config/LJSpeech/train.yaml --duration_control 0.8 --energy_control 0.8
+   ```
+
+   The reusult will be under `output\result\LJSpeech`.
```
Comment (kevintab95) [inline README.md:46]:
```suggestion
   The result will be under `output\result\LJSpeech`.
```

**Review by kevintab95 — APPROVED** [review]
Just a minor typo, but LGTM otherwise. Thanks!

File: README.md [hunk 2]
```diff
@@ -1,5 +1,50 @@
...
+   
+   ```bash
+   cd NEU-tts-2022
+   pip install -r requirements.txt
+   ```
+
+   Install PyTorch
+   
+   ```bash
+   # Intel MacBook
```
Comment (kevintab95) [inline README.md:30]: Can these be applied to other Intel devices too?

Comment (ryangawei) [reply README.md:30]: Yeah I'm using a Windows notebook rn. Since you guys are using MacBook I only put the MacOS commands here

**Pull request merged**